docs/content/paket-pack.md

```diff
@@ -0,0 +1,11 @@
+# paket pack
+Packs all paket.template files within this repository
+
+    [lang=batchfile]
+    $ paket pack output outputDirectory buildconfig Debug
+
+Options:
+
+  `output`: Output directory to put nupkgs
+
+  `buildconfig`: Optionally specify build configuration that should be packaged (defaults to Release).
```

Comment:
$ paket pack output outputDirectory buildconfig Debug

should be 

  $ paket pack output outputDirectory [buildconfig Debug]

to indicate optionality